packages/legacy-scripting-runner/test/example-app/index.js

```diff
@@ -317,6 +317,15 @@ const legacyScriptingSource = `
         return bundle.request;
       },
 
+      movie_pre_poll_double_headers: function(bundle) {
+        bundle.request.url = '${HTTPBIN_URL}/get';
+        bundle.request.headers = {
+          'x-api-key': 'two',
+          'X-API-KEY': 'three'
+        };
+        return bundle.request;
```

Comment:
By the time `movie_pre_poll_double_headers` returns, `bundle.request.headers` only has two entries: `x-api-key: two` and `X-API-KEY: three`.

We expect the final headers should only contain one entry `x-api-key: three`.